### PR TITLE
Small changes to #48

### PR DESCRIPTION
**Pull request by Tech-TTGames**
## Description

Some more consistency across * vs [1]
Swapping Strip buttons and Community support as minor vs. major features

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,52 +5,54 @@ Feel free to check out the full project on [GitHub](https://github.com/Tech-TTGa
...
+<sub>[1] - When a feature is added to the main bot it will be removed from here.<sub>
 
-<sub>[2]*Note: This setup assumes you have the main bot added to the server and configured, Support will not be provided to those who do not assume a likewise configuraton.</sub>
+<sub>[2] - Note: This setup assumes you have the main bot added to the server and configured, Support will not be provided to those who do not assume a likewise configuraton.</sub>
 
 ## Feature 1 - Staff Notes
 
 Private threads are *(now)* free!
 
-This creates a private thread in a channel and adds the staff roles to it*.
```
Comment (kk5dire) [inline README.md:19]: I agree with the changes made in the first footnote sector. 
However I believe that * should be used opposed to <sup>[#]</sup> when there is only one footnote added.

**Review by kk5dire — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kk5dire) [reply README.md:19]: To add, this would also apply to the other sector where only one footnote is added, however I forgot to add a comment to said sector

**Review by kk5dire — APPROVED** [review]
LGTM 👍🏽

**Comment by kk5dire** [comment]
@Tech-TTGames do you want to just add a dummy toml so that way @deepsourcebot dosen't complain on pages, PRs?

**Comment by Tech-TTGames** [comment]
I'll make one in a sec

**Pull request merged**